### PR TITLE
Add distributor and distributor.yaml reference files

### PR DESCRIPTION
**Pull request by StackScribe**
Signed-off-by: Meg McRoberts <meg.mcroberts@dynatrace.com>

This PR creates two reference pages: miscellaneous/distributor and files/distributor.yaml, based on information in the distributor README.md file.

Some of this information overlaps information in the integrations/custom-integration ("Write a Keptn-service") page and, when there are discrepancies between the two, I don't know which is correct.  I would like input on that.  I think the full list of environment variables belongs in the reference page rather than the guide chapter.  Also a question as to whether to present that info in a table (as is done in the guide) or in a simple list (as is currently done in the reference page).

This PR applies these changes to the 0.14.x  and 0.15.x trees.

https://github.com/keptn/keptn.github.io/issues/1094

### CODE REVIEW

File: content/docs/0.14.x/reference/miscellaneous/_index.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+title: Miscellany
```
Comment (oleg-nenashev) [inline content/docs/0.14.x/reference/miscellaneous/_index.md:2]: Have never seen this word in docs before but looks legit

**Review by oleg-nenashev — COMMENTED** [review]
Should it be done for 0.15.x only

File: content/docs/0.14.x/reference/miscellaneous/_index.md (hunk 1 above)
Comment (StackScribe) [reply content/docs/0.14.x/reference/miscellaneous/_index.md:2]: I'm not in love with this term.  "Miscellaneous Information" is an alternative but it's so long...  It's back to the old question about what to call man.5 in fhe Unix man pages...

**Review by oleg-nenashev — CHANGES_REQUESTED** [review]
Sorry, forgot it in my queue. Now we also one for Keptn 15. Text-wise it looks good to me

File: content/docs/0.14.x/reference/files/distributor.yaml/index.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      run: distributor
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        run: distributor
+    spec:
+      containers:
+        - name: distributor
+          image: keptndev/distributor:latest
```
Comment (bacherfl) [inline content/docs/0.14.x/reference/files/distributor.yaml/index.md:30]: let's refer to the latest official 0.14.x image `keptn/distributor:0.14.2` here

File: content/docs/0.15.x/reference/files/distributor.yaml/index.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      run: distributor
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        run: distributor
+    spec:
+      containers:
+        - name: distributor
+          image: keptndev/distributor:latest
```
Comment (bacherfl) [inline content/docs/0.15.x/reference/files/distributor.yaml/index.md:30]: let's refer to the latest official 0.15.x image keptn/distributor:0.15.0 here

File: content/docs/0.15.x/reference/files/index.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+            limits:
+              memory: "128Mi"
+              cpu: "500m"
+          env:
+            - name: PUBSUB_URL
+              value: 'nats://keptn-nats'
+            - name: PUBSUB_TOPIC
+              value: 'sh.keptn.internal.event.some-event'
+            - name: PUBSUB_RECIPIENT
+              value: 'your-service'
```
Comment (bacherfl) [inline content/docs/0.15.x/reference/files/index.md:46]: let's use some domain as an example here to make it clear that this should point towards a http service endpoint

File: content/docs/0.15.x/reference/files/distributor.yaml/index.md (hunk 3 above)
Comment (StackScribe) [reply content/docs/0.15.x/reference/files/distributor.yaml/index.md:30]: done

File: content/docs/0.14.x/reference/files/distributor.yaml/index.md (hunk 2 above)
Comment (StackScribe) [reply content/docs/0.14.x/reference/files/distributor.yaml/index.md:30]: done

**Review by bacherfl — APPROVED** [review]

**Review by oleg-nenashev — APPROVED** [review]

**Pull request merged**